packages/frontend-v2/components/Header/AccountOverview.tsx

```diff
@@ -0,0 +1,147 @@
+import {
+  useDisclosure,
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  Text,
+  Flex,
+  FormControl,
+  Input,
+  FormLabel,
+} from "@chakra-ui/react";
+import { API } from "@graduate/api-client";
+import { StudentModel } from "@graduate/common";
+import { useState } from "react";
+import { toast } from "../../utils";
+import { KeyedMutator } from "swr";
+
+interface AccountOverviewProps {
+  student: StudentModel<string>;
+  mutateStudent: KeyedMutator<StudentModel<string>>;
+}
+export const AccountOverview: React.FC<AccountOverviewProps> = ({
```

Comment:
Should we still use react hook forms here? There is no DTO for changing the full name so it would be a little bit nasty to be able to put this together but if you think its worth having everything the same, then we can do it